src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-import-dialog/tutorial-groups-registration-import-dialog.component.ts

```diff
@@ -259,6 +259,30 @@ export class TutorialGroupsRegistrationImportDialogComponent implements OnInit,
         });
     }
 
+    generateCSV(example: number) {
+        let csvContent: string;
+        switch (example) {
+            case 1:
+                csvContent = 'data:text/csv;charset=utf-8,Title\n';
+                break;
+            case 2:
+                csvContent = 'data:text/csv;charset=utf-8,Title,Matriculation Number,First Name,Last Name\n';
+                break;
+            case 3:
+                csvContent = 'data:text/csv;charset=utf-8,Title,Login,First Name,Last Name\n';
+                break;
+            default:
+                csvContent = '';
+        }
+        const encodedUri = encodeURI(csvContent);
+        const link = document.createElement('a');
+        link.setAttribute('href', encodedUri);
+        link.setAttribute('download', `example${example}.csv`);
+        document.body.appendChild(link);
+        link.click();
```

Comment:
Is that actually how downloads should be handled or is that a quick and dirty fix?

Reply:
This was the approach i knew. It is not intended to be a quick or dirty fix 👍